frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -520,7 +551,12 @@ export async function streamResult(
   }
 }
 
-export async function streamResults(url: string, cb: StreamResultsCb, errCb: StreamErrCb) {
+export async function streamResults(
+  url: string,
+  queryParams: QueryParameters | undefined,
```

Comment:
Same about breaking API.

Reply:
Indeed!

Reply:
Has moved the queryParams to the last position!